cmd/main.go

```diff
@@ -2,13 +2,21 @@ package main
 
 import (
 	"github.com/CESARBR/knot-cloud-storage/internal/config"
+	"github.com/CESARBR/knot-cloud-storage/pkg/data"
 	"github.com/CESARBR/knot-cloud-storage/pkg/server"
 
 	"github.com/CESARBR/knot-cloud-storage/pkg/logging"
 )
 
+var dao = data.DataDAO{}
```

Comment:
If we use `dataInteractor` in some [files](https://github.com/CESARBR/knot-cloud-storage/pull/15/commits/ee910cae761bb02b9bf4d5aa40f034732c796032#diff-d0a0d278de20ac831dd91f3f67034f3dR28) and `dao` in others (such as this), it might get a bit confusing :stuck_out_tongue: 
I suggest we standardize this to `xxxInteractor` (such as `dataInteractor`) in all files :grimacing:

Reply:
Done!